config.toml

```diff
@@ -0,0 +1,131 @@
+# This file shows the default configuration of Meilisearch.
+# All variables are defined here https://docs.meilisearch.com/learn/configuration/instance_options.html#environment-variables
+
+# db_path = "./data.ms"
+# The destination where the database must be created.
+
+# env = "development" # Possible values: [development, production]
+# This environment variable must be set to `production` if you are running in production.
+# More logs wiil be displayed if the server is running in development mode. Setting the master
+# key is optional; hence no security on the updates routes. This
+# is useful to debug when integrating the engine with another service.
+
+# http_addr = "127.0.0.1:7700"
+# The address on which the HTTP server will listen.
+
+# master_key = "MASTER_KEY"
+# Sets the instance's master key, automatically protecting all routes except GET /health.
+
+# no_analytics = false
+# Do not send analytics to Meilisearch.
+
+# disable_auto_batching = false
+# The engine will disable task auto-batching, and will sequencialy compute each task one by one.
+
+
+### DUMP
+
+# dumps_dir = "dumps/"
+# Folder where dumps are created when the dump route is called.
+
+# import_dump = "./path/to/my/file.dump"
+# Import a dump from the specified path, must be a `.dump` file.
+
+# ignore_missing_dump = false
+# If the dump doesn't exist, load or create the database specified by `db_path` instead.
+
+# ignore_dump_if_db_exists = false
+# Ignore the dump if a database already exists, and load that database instead.
+
+###
+
+
+# log_level = "INFO" # Possible values: [ERROR, WARN, INFO, DEBUG, TRACE]
+# Set the log level.
+
+
+### INDEX
+
+# max_index_size = "100 GiB"
+# The maximum size, in bytes, of the main LMDB database directory.
+
+# max_indexing_memory = "2 GiB"
+# The maximum amount of memory the indexer will use. It defaults to 2/3 of the available
+# memory. It is recommended to use something like 80%-90% of the available memory, no
```

Comment:
Could you please remove this sentence? We don't have any recommendations as it depends on the usage